eng/deploy.yaml

```diff
@@ -56,7 +68,9 @@ stages:
 
   - job: deployMaestro
     displayName: Deploy maestro service fabric application
-    dependsOn: updateDatabase
+    dependsOn: 
+    - updateDatabase
+    - approval
```

Comment:
updateDatabase is already depending on approval, you don't have to add both here

Reply:
I want to be safe.  That's true right now, but if someone changes something else, we don't want this guy to randomly go before approval.